WordPressAuthenticator/Extensions/WPStyleGuide+Login.swift

```diff
@@ -145,43 +142,45 @@ extension WPStyleGuide {
             // Create an attributed string that contains the Google icon.
             let font = WPStyleGuide.mediumWeightFont(forStyle: .subheadline)
             googleAttachment.bounds = CGRect(x: 0,
-                                             y: font.descender + Constants.googleIconHyperlinkOffset,
+                                             y: font.descender + Constants.googleIconOffset,
                                              width: googleIcon.size.width,
                                              height: googleIcon.size.height)
 
             return NSAttributedString(attachment: googleAttachment)
         } else {
             // Create an attributed string that contains the Google icon + button text.
-            googleAttachment.bounds = CGRect(x: 0,
-                                             y: Constants.googleIconButtonOffset,
-                                             width: Constants.googleIconButtonSize,
-                                             height: Constants.googleIconButtonSize)
+            googleAttachment.bounds = CGRect(x: 0, y: (NUXButton.titleFont.capHeight - Constants.googleIconButtonSize) / 2,
+                                            width: Constants.googleIconButtonSize, height: Constants.googleIconButtonSize)
 
             let buttonString = NSMutableAttributedString(attachment: googleAttachment)
-            let googleTitle = NSLocalizedString(" Continue with Google", comment: "Button title. Tapping begins log in using Google. There is a leading space to separate it from the Google logo.")
+            let googleTitle = NSLocalizedString("  Continue with Google", comment: "Button title. Tapping begins log in using Google. There are leading spaces to separate it from the Google logo.")
```

Comment:
Could we please revert this? It changes translatable strings unnecessarily and whitespace around translations is discouraged and stripped from our systems. What about this?
```
let googleTitle = "  " + NSLocalizedString("Continue with Google", comment: "Button title. Tapping begins log in using Google. There are leading spaces to separate it from the Google logo.")
```